Cubical/Structures/Axiom.agda

```diff
@@ -0,0 +1,43 @@
+{-# OPTIONS --cubical --no-import-sorts --safe #-}
+module Cubical.Structures.Axiom where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Equiv
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Isomorphism
+open import Cubical.Foundations.Univalence
+open import Cubical.Foundations.Path
+open import Cubical.Foundations.SIP
+open import Cubical.Data.Sigma
+
+private
+  variable
+    ℓ ℓ₁ ℓ₁' ℓ₂ : Level
+
+-- Now, we want to add axioms (i.e. propositions) to our Structure S that don't affect the ι.
+-- We use a lemma due to Zesen Qian, which can now be found in Foundations.Prelude:
+-- https://github.com/riaqn/cubical/blob/hgroup/Cubical/Data/Group/Properties.agda#L83
```

Comment:
Do we use this lemma?

Reply:
I'm not really sure what it's referring to.

Reply:
I think it's outdated and should be deleted